app/controllers/items_controller.rb

```diff
@@ -0,0 +1,42 @@
+class ItemsController < ApplicationController
+  def index
+  end
+
+  def edit
+  end
+
+  def show
+  end
+
+  def update
+
+  end
+
+  def create
+    @user = User.find(params[:user_id])
+    @item = current_user.items.build( item_params )
+    @item.user = @user
+    @new_item = Item.new
```

Comment:
It looks like you still need to save `@item`, which would explain why you aren't seeing any of the items you see getting added via AJAX after a refresh.